### PR TITLE
Fixed few typos

### PR DESCRIPTION
**Pull request by hemchander23**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,21 +11,21 @@ Aarogya Setu mobile application provides the following features:
 - Minimal and simple user interface, which user can get easily acquainted with
 - Scan nearby Aarogya Setu user using BluetoothLE Scanner 
 - Advertise to nearby Aarogya Setu user using BluetoothLE GATT Server
-- Update user about nearby activity using Location Service
+- Update user about the nearby activity using Location Service
```
Comment (crowded-geek) [inline README.md:14]: Are you sure that this is grammatically correct?

File: README.md [hunk 2]
```diff
@@ -11,21 +11,21 @@ Aarogya Setu mobile application provides the following features:
...
 - Available in 12 different languages
-- Nation wide COVID-19 Statistics
+- Nationwide COVID-19 Statistics
 - Self-Assessment as per MoHFW and ICMR guidelines
 - Emergency Helpline Contact
 - List of ICMR approved labs with COVID-19 testing facilities
 - e-Pass integration
 
-The Aarogya Setu App is being widely used by more than 11 Crore Users. The App has been highly successful in identifying people with high risk of COVID-19 infection and has also played a major role in identifying potential COVID-19 hotspots. In the larger public interest and in order to help the international community in their COVID-19 efforts, the Government of India is opening the source code of this App under Apache License 2.0.
+The Aarogya Setu App is being widely used by more than 11 Crore Users. The App has been highly successful in identifying people with a high risk of COVID-19 infection and has also played a major role in identifying potential COVID-19 hotspots. In the larger public interest and in order to help the international community in their COVID-19 efforts, the Government of India is opening the source code of this App under Apache License 2.0.
```
Comment (crowded-geek) [inline README.md:24]: Are you sure that this is grammatically correct?

File: README.md [hunk 3]
```diff
@@ -11,21 +11,21 @@ Aarogya Setu mobile application provides the following features:
...
 
-If you find any security issues or vulnerabilities in the code, then you can send the details to us at : as-bugbounty@nic.in
+If you find any security issues or vulnerabilities in the code, then you can send the details to us at : [as-bugbounty@nic.in](mailto:as-bugbounty@nic.in)
 
-If you want to convey any other feedback regarding the App or Code, then you can send it to us at : support.aarogyasetu@nic.in
+If you want to convey any other feedback regarding the App or Code, then you can send it to us at : [support.aarogyasetu@nic.in](mailto:support.aarogyasetu@nic.in)
```
Comment (crowded-geek) [inline README.md:28]: These changes are already mentioned in #4

File: README.md [hunk 4]
```diff
@@ -44,7 +44,7 @@ If you want to convey any other feedback regarding the App or Code, then you can
 
 **keystore.properties**
 
-Setup a keystore.properties at the root folder with following sample detail and your configurations
+Setup a keystore.properties file at the root folder with the following sample detail and your configurations
```
Comment (crowded-geek) [inline README.md:47]: Unnecessary

File: README.md (hunk 1 above)
Comment (hemchander23) [reply README.md:14]: Yeah. Ran it through Grammarly to cross-check

File: README.md (hunk 3 above)
Comment (hemchander23) [reply README.md:28]: Thanks! Missed it. Seems like it is taken care

**Comment by hemchander23** [comment]
Closing since most of the typos are taken care of already 👍

File: README.md (hunk 2 above)
Comment (hemchander23) [reply README.md:24]: Yeah. Ran it through Grammarly

**Pull request closed without merging**